.github/workflows/depsreview.yaml

```diff
@@ -13,5 +13,5 @@ jobs:
         with:
           persist-credentials: false
 
-      - name: 'Dependency Review'
+      - name: Dependency Review
```

Comment:
If we start like this, then we should remove all quotes that are unnecessary 😉

Reply:
Well, it's without quotes already in most cases hence the change

Reply:
I mean in all other yaml files 😉